testsuite/openshift/objects/rate_limit.py

```diff
@@ -60,6 +60,6 @@ def _policy_is_ready(obj):
             assert success
 
         # https://github.com/Kuadrant/kuadrant-operator/issues/140
-        sleep(60)
+        sleep(90)
```

Comment:
When I was testing this (had this test on a loop for an hour), I had 1/5 of the tests failing because of an issue mentioned in the comment. When the sleep was increased, I had 100% success. So my suggestion is, to pump this to 90.